opencti-platform/opencti-front/src/utils/hooks/useLocalStorage.ts

```diff
@@ -304,7 +287,7 @@ export const usePaginationLocalStorage = <U>(
                 ...viewStorage.filters,
                 filters: [
                   ...viewStorage.filters.filters
-                    .filter((f) => f.key !== k || f.operator !== op), // remove filter with key=k and operator=op
+                    .filter((f: Filter) => f.key !== k || f.operator !== op), // remove filter with key=k and operator=op
```

Comment:
useless type

Reply:
Here is the reason why I have put types here and there.
This is the warning I got in my IDE.
![image](https://github.com/OpenCTI-Platform/opencti/assets/13460700/0b6be489-3eb0-4ce5-a843-01ef6aeb2eba)

I understood since then the CI doesn't block the implicit `any`.
I remove this last one and check for any remaining one. Thanks.

Reply:
I think there are problems with your IDE, I have no warnings...